package.json

```diff
@@ -45,6 +45,7 @@
     "lodash.debounce": "^4.0.8",
     "moment": "2.27.0",
     "prop-types": "15.7.2",
+    "query-string": "^7.1.1",
```

Comment:
hmm I believe I remember seeing Adam mentoining we prefer URLSearchParams over query-string now, but let me check on that a bit in enterprise-frontend

https://dev.to/nerdyman/replacing-query-string-with-native-urlsearchparams-4kdg#:~:text=query%2Dstring%20is%20an%20awesome,both%20Node%20and%20the%20browser!

Reply:
I'd sawy if URLSearchParams meets your needs, let's use that in favor of adding a library dep